src/modules/rewards/Rewards.constants.ts

```diff
@@ -26,3 +27,27 @@ export const ActivityTypeID = {
     image: Twitter
   }
 };
+
+interface Illustration {
+  JoinURL: string;
+  name: string;
+}
+
+interface Illustrations {
+  [key: string]: Illustration;
+}
+
+export const ActivityConfig: Illustrations = {
```

Comment:
@abhishek-01k why do we use capitalised naming everywhere. It should be camel case `activityConfig`